LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 Viorel Cojocaru <vio@mavrix.es>
```

Comment:
Thank you @phiggins! Can you please change the holder to `Mavrix Technologies` (the organization is the owner of the repository)?

Reply:
Done!